lib/modules/manager/npm/post-update/npm.spec.ts

```diff
@@ -27,16 +27,14 @@ describe('modules/manager/npm/post-update/npm', () => {
     // package.json
     fs.readLocalFile.mockResolvedValueOnce('{}');
     fs.readLocalFile.mockResolvedValueOnce('package-lock-contents');
-    const skipInstalls = true;
-    const postUpdateOptions = ['npmDedupe'];
     const updates = [
       { packageName: 'some-dep', newVersion: '1.0.1', isLockfileUpdate: false },
     ];
     const res = await npmHelper.generateLockFile(
       'some-dir',
       {},
       'package-lock.json',
-      { skipInstalls, postUpdateOptions },
+      {},
```

Comment:
Can these changes be reverted?

Reply:
Sure. But test results would change significantly even than unless we add a NPM version constraint to use NPM 6.

Current test suit misses a test running with default configuration. That's the root cause here. I can refactor the tests in a separate PR which we merge before of you prefer that.